### ISSUE TITLE
Graphing mode: Contents of equation is not read by Narrator.

### ISSUE THREAD

**Issue by joshkoon** [microsoft/calculator#950]
**Describe the bug**
Using Narrator in Graphing mode, Narrator does not announce the contents of the equations.

**Steps To Reproduce**

1. Go to Graphing mode
2. Add an equation, e.g. "x + 2"
3. Navigate to the equation edit box with Narrator.
4. Narrator always reads, "Enter an equation, Edit", regardless of the contents of the edit box.

**Expected behavior**
Users should be able to have the contents of their equations announced to them.

**Screenshots**

**Device and Application Information**
- OS Build: 10.0.19547.0
- Architecture: x64
- Application Version: 10.1912.4.0
- Region: en-US
- Dev Version Installed:

**Additional context**

**Requested Assignment**

I'm just reporting this problem.  I don't want to fix it.

**Comment by MicrosoftIssueBot** [comment]
This is your friendly Microsoft Issue Bot. I've seen this issue come in and have gone to tell a human about it.

**Comment by grochocki** [comment]
@joseartrivera Can you verify whether this is working or not? I was able to get equations read out properly, but we should double check if there is something here.

**Comment by joseartrivera** [comment]
These are rich edit boxes, to get narrator to read them you can use any command that tells narrator to read the contents of the textbox, such as caps lock + down arrow. See [Appendix B: Narrator keyboard commands and touch gestures](https://support.microsoft.com/en-us/help/22806/windows-10-narrator-keyboard-commands-touch-gestures) for more examples.
